### PR TITLE
Update README.md to state this connector does not support write.

### PR DESCRIPTION
**Pull request by cyxxy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 The connector uses the [Spark SQL Data Source API](https://spark.apache.org/docs/latest/sql-programming-guide.html#data-sources) to read data from [Google BigQuery](https://cloud.google.com/bigquery/).
 
+It does not support writing data to [Google BigQuery](https://cloud.google.com/bigquery/).
```
Comment (kmjung) [inline README.md]: s/"It"/"The connector", or else merge this into the paragraph above.

Comment (cyxxy) [reply README.md]: Done. Also added link to the other BQ connector which supports write.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 The connector uses the [Spark SQL Data Source API](https://spark.apache.org/docs/latest/sql-programming-guide.html#data-sources) to read data from [Google BigQuery](https://cloud.google.com/bigquery/).
 
+The connector does not support writing data to [Google BigQuery](https://cloud.google.com/bigquery/). If you want to write data to Google BigQuery, you can use another [BigQuery Connector](https://cloud.google.com/dataproc/docs/concepts/connectors/bigquery), which implements the HDFS interface.
+
```
Comment (functicons) [inline README.md]: It is actually not HDFS interface, but "Apache Hadoop InputFormat and OutputFormat classes"

Comment (cyxxy) [reply README.md]: done.

**Pull request merged**